code/modules/mob/living/carbon/human/human.dm

```diff
@@ -100,7 +100,7 @@
 			var/obj/item/organ/heart/ethereal/eth_heart = getorganslot(ORGAN_SLOT_HEART)
 			if(istype(eth_species))
 				. += "Crystal Charge: [round((nutrition / NUTRITION_LEVEL_MOSTLY_FULL) * 100, 0.1)]%"
-			if(eth_heart)
+			if(eth_heart && istype(eth_heart, /obj/item/organ/heart/ethereal))
```

Comment:
You don't actually need the second argument for istype() here because you already defined eth_heart as that type